psu/imx6ull-ram.sdp

```diff
@@ -0,0 +1,18 @@
+# Wait for device
+WAIT 0x15a2 0x80
+ERROR_STATUS
+
+# Load kernel part with DCD initializing DDR
+WRITE_FILE F "phoenix-armv7a7-imx6ull.img" 0 0x00907000 0x1000
+# Jump to ivt_addr
+JUMP_ADDRESS 0x00907400
+
+# Load the rest of kernel
+WAIT 0x15a2 0x80
+ERROR_STATUS
+
+# Write whole image
+WRITE_FILE F "phoenix-armv7a7-imx6ull.img" 0 0x80000000
+
+# Run kernel
+JUMP_ADDRESS 0x80000728
```

Comment:
In all other scripts the jump adress is `0x80000718`. Are You sure it's correct?

Reply:
Ack. Please take care when merging into DTR project (update also SDP scripts in `_boot` dir.